### PR TITLE
FIX: DatatableQueryBuilder is using alias which can be potently a reserved word. #735

### PR DESCRIPTION
**Pull request by jojo1981**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,14 @@
         "doctrine/orm": "^2.5",
         "symfony/options-resolver": "^3.0|^4.0",
         "symfony/property-access": "^3.0|^4.0",
-        "friendsofsymfony/jsrouting-bundle": "^1.6|^2.0"
+        "symfony/translation": "^3.0|^4.0",
+        "symfony/security": "^3.0|^4.0",
+        "sensio/generator-bundle": "^3.1",
+        "friendsofsymfony/jsrouting-bundle": "^1.6|^2.0",
+        "twig/twig": "^2.4"
```
Comment (Seb33300) [inline composer.json]: Why did you add all of this dependencies?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/composer.lock
+/vendor
```
Comment (Seb33300) [inline .gitignore]: Can you please remove this file

Comment (jojo1981) [reply .gitignore]: OK I don't know why, but I will remove this file.

File: composer.json (hunk 1 above)
Comment (jojo1981) [reply composer.json]: This are the dependencies who are missing and on which this library depends.

**Comment by jojo1981** [comment]
Now with passed build (also for hvm).

File: composer.json (hunk 1 above)
Comment (jojo1981) [reply composer.json]: I will remove this, because it's no part of this PR (issue) and should be fixed in a separate issue.

**Comment by Seb33300** [comment]
Thanks 👍

**Comment by jojo1981** [comment]
Thx!
